mods/ra/maps/soviet-soldier-volkov-n-chitzkoi/rules.yaml

```diff
@@ -0,0 +1,233 @@
+World:
+	LuaScript:
+		Scripts: scu35ea.lua, scu35ea-AI.lua
+	MissionData:
+		Briefing: The Allies have stolen a new armor plating from us. With it, they can make structures nearly invulnerable. Their processing plant must be destroyed.\n\nFor this critical mission, we are allowing you access to Soldier Volkov. Use him and his cybernetic dog to sabotage the Allied control center, allowing our units in the southeast access to the alloy facility.\n\nThe mission is a success when Volkov destroys the control center, and our units destroy the alloy facility.\n
+		WinVideo: grvestne.vqa
+		LossVideo: sovcemet.vqa
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+
+Player:
+	PlayerResources:
+		DefaultCash: 0
+	LobbyPrerequisiteCheckbox@GLOBALBOUNTY:
+		Enabled: False
+		Locked: True
+
+MSLO:
+	Buildable:
+		Prerequisites: ~disabled
+        
+PROC:
+	-FreeActor:
+        
+E7:
+	Buildable:
+		Prerequisites: ~disabled
+	Health:
+		HP: 25000
+	Armor:
+		Type: Heavy
+        
+2TNK:
+	Buildable:
+		Prerequisites: ~vehicles.allies, ~techlevel.medium
+        
+CA:
+	Buildable:
+		Prerequisites: ~syrd, ~techlevel.high
+        
+C2:
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
```

Comment:
Have you tried defining `Inherits@2: ^ArmedCivilian` instead?  This would save a lot of duplication.

Reply:
Yeah I'll upload the new file soon